debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -311,6 +311,10 @@ private static void dumpModifiers(PrintStream out, long modifiers, int modifierS
 					out.append("(constant) ");
 					modifiers &= ~J9FieldFlags.J9FieldFlagConstant;
 				}
+				if ((modifiers & J9FieldFlags.J9FieldFlagIsNullRestricted) != 0) {
+					out.append("(NullRestricted) ");
+					modifiers &= ~J9FieldFlags.J9FieldFlagIsNullRestricted;
+				}
```

Comment:
I think this is only appropriate for instance fields; `J9StaticFieldIsNullRestricted` (which has a different value than `J9FieldFlagIsNullRestricted`) should be checked for static fields.

Reply:
That is for RAM classes, for ROM classes they both use `modifiers`.

Reply:
We're talking about fields here, not classes, but to confirm, all null-restricted fields have `J9FieldFlagIsNullRestricted` included in `J9ROMFieldShape.modifiers` independent of whether the field is static or not?

Reply:
Yes. All fields written into the ROM class will have `J9FieldFlagIsNullRestricted ` in modifiers, the `J9StaticFieldIsNullRestricted ` field applies to resolved fields.